NN_2L2H2O_poly2d/NN_2L2H2O_poly2d_benchmarking_result.md

```diff
@@ -0,0 +1,23 @@
+# Benchmarking result of NN_2L2OH2O_poly2d 
+
+### This file contains :  
+
+#### 1. the benchmarking result of `NN_2L2OH2O_poly2d` in Neural Network model on 42105 samples. This benchmarking is made to the part of the code *predicting the final score* of all samples, excluding *loading parameters, initializing layers and GPU resource context, etc*. The result records the run time on different machine/GPU, supporting library version, driver version, compiler version etc. 
```

Comment:
don't use the title marker `####` for such big amount of text, normal text is fine.